test/fixtures/run-script/package.json

```
@@ -10,6 +10,8 @@
     "path-env": "path-env",
     "path-env-windows": "path-env.bat",
     "special-env-variables": "special-env-variables",
-    "special-env-variables-windows": "special-env-variables.bat"
+    "special-env-variables-windows": "special-env-variables.bat",
+    "cd": "cd",
+    "pwd": "pwd"
```

Lets use the naming, just like the rest of the scripts

```suggestion
    "cd-windows": "pwd"
```